cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -101,13 +101,12 @@ <h2>Related Datasets</h2>
     {% if record.distribution and (record.distribution.number_events or record.distribution.number_files or record.distribution.size) %}
         <div class="row">
             <div class="col-md-12">
-                <h2> Characteristics </h2>
-                <label> Dataset: </label>
+                <h2>Dataset Characteristics </h2>
```

Comment:
We cannot use `Dataset` here, since the template is used for all types of records. E.g. see http://opendata.cern.ch/record/250

Reply:
But `Dataset` always appears in the template, even for the VM Image: `Dataset: 1 files 15.2 MB in total`

Reply:
Yes, it is better to say "Data:" there, which is more generic... or perhaps even nothing at all and start simply with "25465895 events 2814 files 2.6 TB in total".